### PR TITLE
Create Indonesian translation

### PR DESCRIPTION
**Pull request by Nicholas2750**

### CODE REVIEW

**Comment by ddevault** [comment]
Thanks! It might take a while to find someone to review your work, though...

File: README.id.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
+# sway
+
+[**English**](https://github.com/swaywm/sway/blob/master/README.md#sway--) - [日本語](https://github.com/swaywm/sway/blob/master/README.ja.md#sway--) - [Deutsch](https://github.com/swaywm/sway/blob/master/README.de.md#sway--) - [Ελληνικά](https://github.com/swaywm/sway/blob/master/README.el.md#sway--) - [Français](https://github.com/swaywm/sway/blob/master/README.fr.md#sway--) - [Українська](https://github.com/swaywm/sway/blob/master/README.uk.md#sway--) - [Indonesian](https://github.com/swaywm/sway/blob/master/README.id.md#sway--) - [Italiano](https://github.com/swaywm/sway/blob/master/README.it.md#sway--) - [Português](https://github.com/swaywm/sway/blob/master/README.pt.md#sway--) -
+[Русский](https://github.com/swaywm/sway/blob/master/README.ru.md#sway--) - [Български](https://github.com/swaywm/sway/blob/master/README.bg.md#sway--)
+
+sway adalah komposer untuk i3 [Wayland](http://wayland.freedesktop.org/).
```
Comment (x8BitRain) [inline README.id.md]: Replace komposer with compositor, there isn't an Indonesian word for Compositor and komposer isn't the right word to describe it so you'd default to the English word for it.

File: README.id.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+Pertanyaaan dan jawaban umum [FAQ](https://github.com/swaywm/sway/wiki). Ikut [channel
```
Comment (x8BitRain) [inline README.id.md]: Untuk pertanyaan dan jawaban ikut IRCnya.

File: README.id.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+ IRC](http://webchat.freenode.net/?channels=sway&uio=d4) (#sway on
+irc.freenode.net).
+
+Jika ingin mendukung perkembangan sway, kunjungi [Halaman Pateron SirCmpwn](https://patreon.com/sircmpwn).
```
Comment (x8BitRain) [inline README.id.md]: Kalau mau mendukung proyek ini, bikin kontribusi di Patreon [link]

File: README.id.md [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+
+## Tanda Tangan Release
```
Comment (x8BitRain) [inline README.id.md]: Just say "Signature Rilis" as Tanda Tangan means hand written signature.

File: README.id.md [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+
+Tiap Release ditandatangani dengan [B22DA89A](http://pgp.mit.edu/pks/lookup?op=vindex&search=0x52CB6609B22DA89A)
```
Comment (x8BitRain) [inline README.id.md]: Semua rilis tertanda dengan B22DA89A dan menerbitkan [di github]

File: README.id.md [hunk 6]
```diff
@@ -0,0 +1,71 @@
...
+dan diterbitkan [di GitHub](https://github.com/swaywm/sway/releases).
+
+## Instalasi
+
+### Dari Packages
+
+Sway tersedia dalam beberapa distribusi. Lakukan instalasi dengan package "sway" terlebih dahulu. 
+Apabila gagal, lihatlah [halaman wiki](https://github.com/swaywm/sway/wiki/Unsupported-packages)
+untuk informasi instalasi untuk tiap distribusi.
+
```
Comment (x8BitRain) [inline README.id.md]: Sway tersedia untuk banyak distibusi Linux, coba di instal package "sway" untuk distibusi anda, kalau tidak tersedia untuk distibusi anda, coba baca [Wiki] untuk informasi menginstal di distibusi anda.

File: README.id.md [hunk 7]
```diff
@@ -0,0 +1,71 @@
...
+Jika anda tertarik membuat package untuk distribusi anda, kunjungi channel IRC
```
Comment (x8BitRain) [inline README.id.md]: Jika anda mau membuat package untuk distribusi anda, kunjungi channel IRC

File: README.id.md [hunk 8]
```diff
@@ -0,0 +1,71 @@
...
+atau kirim email ke sir@cmpwn.com untuk saran.
+
+### Kompilasi dari Sumber
```
Comment (x8BitRain) [inline README.id.md]: Compile dari Source

File: README.id.md [hunk 9]
```diff
@@ -0,0 +1,71 @@
...
+
+Ketergantungan dalam instaasi:
```
Comment (x8BitRain) [inline README.id.md]: Instal dependency

File: README.id.md [hunk 10]
```diff
@@ -0,0 +1,71 @@
...
+* wayland-protocols \*
+* pcre
+* json-c
+* pango
+* cairo
+* gdk-pixbuf2 \*\*
+* [scdoc](https://git.sr.ht/~sircmpwn/scdoc) >= 1.8.1 (optional: man pages) \*
+* git \*
+
+_\*Ketergantungan kompilasi
```
Comment (x8BitRain) [inline README.id.md]: Waktu Compile Dependancy

File: README.id.md [hunk 11]
```diff
@@ -0,0 +1,71 @@
...
+
+_\*\*Tidak wajib: hanya diperlukan untuk swaybg_
+
+Jalankan commands dibawah:
```
Comment (x8BitRain) [inline README.id.md]: Mengetik command ini

File: README.id.md [hunk 12]
```diff
@@ -0,0 +1,71 @@
...
+
+    meson build
+    ninja -C build
+    sudo ninja -C build install
+
+Untuk sistem tanpa logind, jalankan:
```
Comment (x8BitRain) [inline README.id.md]: Untuk sistem tanpa logind, pakai:

File: README.id.md [hunk 13]
```diff
@@ -0,0 +1,71 @@
...
+
+    sudo chmod a+s /usr/local/bin/sway
+
+Sway akan membuang permisi root setelah startup.
+
+## Konfigurasi
+
+Jika anda sudah menggunakan i3, duplikatkan konfigurasi i3 anda ke `~/.config/sway/config` dan sway akan langsung bekerja.
```
Comment (x8BitRain) [inline README.id.md]: Jika anda sudah menggunakan i3, duplikatkan konfigurasi i3 anda ke `~/.config/sway/config` dan sway akan langsung siap.

File: README.id.md [hunk 14]
```diff
@@ -0,0 +1,71 @@
...
+Untuk sistem tanpa logind, jalankan:
+
+    sudo chmod a+s /usr/local/bin/sway
+
+Sway akan membuang permisi root setelah startup.
+
+## Konfigurasi
+
+Jika anda sudah menggunakan i3, duplikatkan konfigurasi i3 anda ke `~/.config/sway/config` dan sway akan langsung bekerja. 
+Jika anda belum menggunakan i3, duplikatkan konfigurasi sampel ke
```
Comment (x8BitRain) [inline README.id.md]: Jika anda belum menggunakan i3, duplikatkan contoh konfigurasi ke

File: README.id.md [hunk 15]
```diff
@@ -0,0 +1,71 @@
...
+`~/.config/sway/config`. Sampel dapat ditemukan di `/etc/sway/config`.
```
Comment (x8BitRain) [inline README.id.md]: ~/.config/sway/config`. contoh konfigurasi bias ada di `/etc/sway/config`

File: README.id.md [hunk 16]
```diff
@@ -0,0 +1,71 @@
...
+Jalankan `man 5 sway` untuk informasi mengenai konfigurasi.
```
Comment (x8BitRain) [inline README.id.md]: Pakai `man 5 sway` untuk informasi mengenai konfigurasi.

File: README.id.md (hunk 9 above)
Comment (ha1zum) [reply README.id.md]: lebih tepat kalau "Dependensi Instalasi"

File: README.id.md (hunk 8 above)
Comment (ha1zum) [reply README.id.md]: Kompilasi dari kode Sumber

File: README.id.md (hunk 10 above)
Comment (ha1zum) [reply README.id.md]: Better: Dependensi saat kompilasi

File: README.id.md (hunk 11 above)
Comment (ha1zum) [reply README.id.md]: Alternative: 
Mengetik perintah ini

File: README.id.md (hunk 6 above)
Comment (ha1zum) [reply README.id.md]: Agree with @x8BitRain version, but there are grammatical errors. I think it should be:

Sway tersedia untuk banyak distribusi Linux, coba instal package "sway" untuk distribusi Anda, kalau tidak tersedia, coba baca [Wiki] untuk informasi instalasi di distribusi Anda.

Alternatively, if "Anda" is too formal, just use "kamu". If you want to use "Anda", that word must be capitalized.

**Comment by ddevault** [comment]
Thanks for the reviews!

File: README.id.md (hunk 1 above)
Comment (Nicholas2750) [reply README.id.md]: Fixed, thanks!

File: README.id.md (hunk 3 above)
Comment (Nicholas2750) [reply README.id.md]: Fixed, thanks!

File: README.id.md (hunk 2 above)
Comment (Nicholas2750) [reply README.id.md]: Fixed, thanks!

File: README.id.md (hunk 4 above)
Comment (Nicholas2750) [reply README.id.md]: Fixed, thanks!

File: README.id.md (hunk 5 above)
Comment (Nicholas2750) [reply README.id.md]: Fixed, thanks!

File: README.id.md (hunk 6 above)
Comment (Nicholas2750) [reply README.id.md]: Went with, ha1zum's suggestion. Also changed all "anda" to"Anda". Thanks!

File: README.id.md (hunk 7 above)
Comment (Nicholas2750) [reply README.id.md]: Fixed, thanks!

File: README.id.md (hunk 8 above)
Comment (Nicholas2750) [reply README.id.md]: Fixed, thanks!

File: README.id.md (hunk 9 above)
Comment (Nicholas2750) [reply README.id.md]: Fixed, thanks!

File: README.id.md (hunk 10 above)
Comment (Nicholas2750) [reply README.id.md]: Fixed, thanks!

File: README.id.md (hunk 11 above)
Comment (Nicholas2750) [reply README.id.md]: Fixed, thanks!

File: README.id.md (hunk 12 above)
Comment (Nicholas2750) [reply README.id.md]: Fixed, thanks!

File: README.id.md (hunk 13 above)
Comment (Nicholas2750) [reply README.id.md]: Fixed, thanks!

File: README.id.md (hunk 14 above)
Comment (Nicholas2750) [reply README.id.md]: Fixed, thanks!

File: README.id.md (hunk 15 above)
Comment (Nicholas2750) [reply README.id.md]: Fixed, thanks!

File: README.id.md (hunk 16 above)
Comment (Nicholas2750) [reply README.id.md]: Fixed, thanks!

**Comment by ddevault** [comment]
Please address the merge conflicts, then I'll pull this in. Thanks!

**Comment by emersion** [comment]
Can you rebase against master and squash all commits into a single one?

**Comment by emersion** [comment]
Feel free to re-open this PR once rebased against master,

**Pull request closed without merging**